arrow/parser.py

```diff
@@ -228,6 +232,8 @@ def parse(self, datetime_string, fmt):
         for token in fmt_tokens:
             if token == "Do":
                 value = match.group("value")
+            elif token == "W":
+                value = (match.group(3), match.group(5), match.group(7))
```

Comment:
Might be good to change this to a named group (ref: https://docs.python.org/3/howto/regex.html#non-capturing-and-named-groups).